interface/ceed.c

```diff
@@ -323,7 +323,7 @@ int CeedInit(const char *resource, Ceed *ceed) {
   }
   if (!matchlen) return CeedError(NULL, 1, "No suitable backend");
   ierr = CeedCalloc(1,ceed); CeedChk(ierr);
-  if (!strcmp(getenv("CEED_ERROR_HANDLER"), "exit"))
+  if (getenv("CEED_ERROR_HANDLER") && !strcmp(getenv("CEED_ERROR_HANDLER"), "exit"))
```

Comment:
Better to assign to a variable to avoid calling `getenv` twice. It'll also be cleaner if we need to test for other names.

Reply:
Updated